superset/views/core.py

```diff
@@ -734,7 +734,7 @@ def explore(  # pylint: disable=too-many-locals,too-many-return-statements,too-m
 
             # if feature enabled, run some health check rules for sqla datasource
             if hasattr(datasource, "health_check"):
```

Comment:
Wouldn't this be always false because you remove the `health_check` method/attribute from `SqlaDatasource`?